cryptoinvestor/views/assets.py

```diff
@@ -42,6 +45,112 @@ def get_objects(self):
                 }
             }
 
+        def dispatch_request(self):
```

Comment:
What is this?

Reply:
It was only test, but I didn't remove it.